### PR TITLE
Add E2E tests for required property on CheckboxV1 and RadioGroupV1

### PR DESCRIPTION
**Pull request by ksiler**
### Platforms Impacted
- [ ] iOS
- [ ] macOS
- [x] win32 (Office)
- [ ] windows
- [ ] android

### Description of changes

Adds E2E tests for the required property on CheckboxV1 and RadioGroupV1

### Verification

Ran `yarn e2etest` locally

| Before                                       | After                                      |
|----------------------------------------------|--------------------------------------------|
| Screenshot or description before this change | Screenshot or description with this change |

### Pull request checklist

This PR has considered (when applicable):
- [x] Automated Tests
- [ ] Documentation and examples
- [ ] Keyboard Accessibility
- [ ] Voiceover
- [ ] Internationalization and Right-to-left Layouts

### CODE REVIEW

File: apps/E2E/src/CheckboxV1/specs/CheckboxV1.spec.win.ts [hunk 1]
```diff
@@ -53,6 +53,22 @@ describe('CheckboxV1 Accessibility Testing', () => {
 
     await expect(await CheckboxV1PageObject.didAssertPopup()).toBeFalsy(CheckboxV1PageObject.ERRORMESSAGE_ASSERT);
   });
+
+  it('Set "required" prop. Validate "IsRequiredForForm" element attribute is true.', async () => {
+    await expect(
+      await CheckboxV1PageObject.compareAttribute(CheckboxV1PageObject._secondaryComponent, Attribute.IsRequiredForForm, 'True'),
```
Comment (samuelfreiberg) [inline apps/E2E/src/CheckboxV1/specs/CheckboxV1.spec.win.ts:59]: I believe "True" can be AttributeValue.true (for consistency across the package and type checking)

File: apps/E2E/src/CheckboxV1/specs/CheckboxV1.spec.win.ts [hunk 2]
```diff
@@ -53,6 +53,22 @@ describe('CheckboxV1 Accessibility Testing', () => {
...
+    ).toBeTruthy();
+
+    await expect(await CheckboxV1PageObject.didAssertPopup()).toBeFalsy(CheckboxV1PageObject.ERRORMESSAGE_ASSERT);
+  });
+
+  it('Do NOT set "required" prop. Validate "IsRequiredForForm" element attribute is false.', async () => {
+    await expect(
+      await CheckboxV1PageObject.compareAttribute(CheckboxV1PageObject._primaryComponent, Attribute.IsRequiredForForm, 'False'),
```
Comment (samuelfreiberg) [inline apps/E2E/src/CheckboxV1/specs/CheckboxV1.spec.win.ts:67]: Same as above

File: apps/E2E/src/RadioGroupV1/specs/RadioGroupV1.spec.win.ts [hunk 3]
```diff
@@ -99,6 +99,22 @@ describe('RadioGroupV1/RadioV1 Accessibility Testing', () => {
 
     expect(await RadioGroupV1PageObject.didAssertPopup()).toBeFalsy(RadioGroupV1PageObject.ERRORMESSAGE_ASSERT);
   });
+
+  it('Set "required" prop on RadioGroup. Validate "IsRequiredForForm" element attribute is true.', async () => {
+    await expect(
+      await RadioGroupV1PageObject.compareAttribute(RadioGroupV1PageObject._secondaryComponent, Attribute.IsRequiredForForm, 'True'),
```
Comment (samuelfreiberg) [inline apps/E2E/src/RadioGroupV1/specs/RadioGroupV1.spec.win.ts:105]: Same as above

File: apps/E2E/src/CheckboxV1/specs/CheckboxV1.spec.win.ts (hunk 1 above)
Comment (ksiler) [reply apps/E2E/src/CheckboxV1/specs/CheckboxV1.spec.win.ts]: Thanks! Didn't know it was available. Updated both spec files.

**Review by samuelfreiberg — APPROVED** [review]

**Pull request merged**